cpg-analysis/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/EvaluateExtensions.kt

```diff
@@ -36,3 +42,40 @@ val NewArrayExpression.capacity: Int
     get() {
         return dimensions.first().evaluate() as Int
     }
+
+/**
+ * A little helper function to find a [CallExpression]s argument first by [name] and if this fails
+ * by [position]. The argument ist evaluated and the result is returned if it has the expected type
+ * [T].
+ *
+ * @param this The [CallExpression] to analyze.
+ * @param name Optionally: the [CallExpression.arguments] name.
+ * @param position Optionally: the [CallExpression.arguments] position.
+ * @param evaluator The [ValueEvaluator] to use for evaluation of the argument.
+ * @return The evaluated result (of type [T]) or `null`.
+ */
+inline fun <reified T> CallExpression.getArgumentValueByNameOrPosition(
```

Comment:
```suggestion
inline fun <reified T> CallExpression.argumentValueByNameOrPosition(
```

would be more Kotlin-like in my opinion